madmom/audio/ffmpeg.py

```diff
@@ -295,7 +295,10 @@ def get_file_info(infile, cmd='ffprobe'):
         if line.startswith(b'channels='):
             info['num_channels'] = int(line[len('channels='):])
         if line.startswith(b'sample_rate='):
-            info['sample_rate'] = int(line[len('sample_rate='):])
+            if cmd == 'avprobe':
+                info['sample_rate'] = int(float(line[len('sample_rate='):]))
```

Comment:
+1 for adding an explicit comment. If it works for avprobe, just go ahead and change it, ffmpeg gets tested automatically.